packages/gravity-ui-nunjucks/src/_patterns/03-organisms/00-page-structure/04-page-intro.md

```diff
@@ -0,0 +1,5 @@
+## Purpose
+
+This is the component for use as the page heading.
+
+It consists of a `<h1>` as the page title, a `<p>` for a short description of the page, an optional `<time>` to display a date related to this content (usually publication date), and an optional `<img>` which will automatically gain a downward pointing notch.
```

Comment:
Maybe add some guidance here about when to use a `<header>` element or a `<div>` to markup the page intro.